### PR TITLE
feat(curriculum): create CSS Grid quiz

### LINKED ISSUE #56194: [HACKTOBERFEST 2024] - CSS Grid Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the CSS Grid quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-css-grid`.

**All quiz questions need to relate to one of these topics:**

- what is css grid
- common grid properties
  - grid-template-columns and rows
  - grid-column and row
- how to use tools like the devtools to debug your css

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by Sebastian-Wlo**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56194

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 1]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+Defines a size range between the first and second value.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the shorthand for `column_gap` and `row_gap` properties?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:130]:
```suggestion
What is the shorthand for the `column-gap` and `row-gap` properties?
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 2]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`display: grid`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which one of these options for `grid-template-columns` property will result in a middle column that is 80 characters wide, and two columns taking up the remaining space?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:64]:
```suggestion
Which one of these options for `grid-template-columns` property will result in a middle column that is 80 relative units wide, and two columns taking up the remaining space?
```

`1rem` isn't necessarily the width of a single character.

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 3]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+Specifies the number and heights of the rows in a grid layout.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does `minmax()` function do?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:108]:
```suggestion
What does the `minmax()` function do?
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 4]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`grid-row-start`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property sets a size for the rows in a grid container?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:174]:
```suggestion
Which property automatically allocates rows based on a fixed size?
```

Technically more accurate - `grid-row` can also set the size.

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 5]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`grid-column: 1 / -1`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does `grid-auto-flow` property do?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:284]:
```suggestion
What does the `grid-auto-flow` property do?
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 6]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --text--
 
-Placeholder question
+Which one of these options is NOT a unit used for the `gap` property?
 
 #### --distractors--
 
-Placeholder distractor 1
+px
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:332]:
```suggestion
`px`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 7]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 2
+%
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:336]:
```suggestion
`%`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 8]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+em
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:340]:
```suggestion
`em`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 9]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --answer--
 
-Placeholder answer
+fr
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:344]:
```suggestion
`fr`
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by Sebastian-Wlo — COMMENTED** [review]
Changes applied - I honestly expected more doubts about the quality of the questions.

**Review by naomi-lgbt — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md [hunk 10]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`display: grid`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which one of these options for `grid-template-columns` property will result in a middle column that is 80 relative units wide, and two columns taking up the remaining space?
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md:64]:
```suggestion
Which one of these options for the `grid-template-columns` property will result in a middle column that is 80 relative units wide, and two columns taking up the remaining space?
```

**Review by majestic-owl448 — APPROVED** [review]
Thank you for your PR

**Pull request merged**